api/room_resource/schema.py

```diff
@@ -197,16 +197,30 @@ def resolve_all_resources(self, info, **kwargs):
         resp = PaginatedResource(**kwargs)
         return resp
 
+    @Auth.user_roles('Admin')
     def resolve_get_resources_by_room_id(self, info, room_id):
         # Get resources of a specifics room
```

Comment:
Kindly correct this from `specifics` to `specific` since it involves the method you are using.

Reply:
This has been fixed. Thanks for the feedback @TeamoreA